app/assets/javascripts/admin/addon/components/admin-user-field-item.js

```diff
@@ -47,9 +47,10 @@ export default Component.extend(bufferedProperty("userField"), {
     "userField.editable",
     "userField.required",
     "userField.show_on_profile",
-    "userField.show_on_user_card"
+    "userField.show_on_user_card",
+    "userField.searchable"
```

Comment:
One trick you can use here to avoid this endless list of tracked properties:

```javascript
@computed("userField.{editable,required,show_on_profile,show_on_user_card,searchable"}
flags(userField) {
  // userField.required ...
}
```